src/PostgREST/Network.hs

```diff
@@ -1,17 +1,13 @@
 module PostgREST.Network
-  ( resolveHost
+  ( resolveAddress
   ) where
 
-import           Data.IP        (fromHostAddress, fromHostAddress6)
 import           Data.String    (IsString (..))
 import qualified Network.Socket as NS
 
 import Protolude
 
-resolveHost :: NS.Socket -> IO (Maybe Text)
-resolveHost sock = do
+resolveAddress :: NS.Socket -> IO Text
+resolveAddress sock = do
   sn <- NS.getSocketName sock
-  case sn of
-    NS.SockAddrInet _ hostAddr ->  pure $ Just $ fromString $ show $ fromHostAddress hostAddr
-    NS.SockAddrInet6 _ _ hostAddr6 _ -> pure $ Just $ fromString $ show $ fromHostAddress6 hostAddr6
-    _ -> pure Nothing
+  return $ fromString $ show sn -- Example: 0.0.0.0:80 or [2001:db8::1]:80
```

Comment:
One thing that changes with this PR, is that previously we displayed the `IPv6` address _slightly_ wrong, it was like:
```
28/Aug/2025:23:46:41 +0500: Admin server listening on ::1:3001
```
Now, the correct way, it's like:
```
28/Aug/2025:23:46:41 +0500: Admin server listening on [::1]:3001
```
@steve-chavez I think this is a small but separate improvement. How about we do this in a separate PR? We don't have a test for it, so can add that too.

Reply:
> How about we do this in a separate PR? We don't have a test for it, so can add that too.

@taimoorzaeem Nice catch. Yes, please go ahead.